### PR TITLE
Chat with us issue fixed on android

### PR DESCRIPTION
**Pull request by VaishakMB08**

### CODE REVIEW

File: lib/src/web_view_stack.dart [hunk 1]
```diff
@@ -36,10 +37,20 @@ class _WebViewStackState extends State<WebViewStack> {
               loadingPercentage = 100;
             });
           },
-          onNavigationRequest: (navigation) {
+          onNavigationRequest: (navigation) async {
+            final Uri uri = Uri.parse(navigation.url);
             if (navigation.url.contains('fb://page/')) {
               return NavigationDecision.prevent;
             }
+            if (navigation.url.contains('tel')) {
```
Comment (nikitharajexathought) [inline lib/src/web_view_stack.dart]: move tel as a constant please

File: lib/src/web_view_stack.dart [hunk 2]
```diff
@@ -36,10 +37,20 @@ class _WebViewStackState extends State<WebViewStack> {
...
+              if (await canLaunchUrl(uri)) {
+                await launchUrl(uri);
+                return NavigationDecision.prevent;
+              } else {
+                print("Cannot launch url");
```
Comment (nikitharajexathought) [inline lib/src/web_view_stack.dart]: why do we need this print statement?

**Review by nikitharajexathought — CHANGES_REQUESTED** [review]

File: lib/src/web_view_stack.dart (hunk 1 above)
Comment (VaishakMB08) [reply lib/src/web_view_stack.dart]: made changes

File: lib/src/web_view_stack.dart (hunk 2 above)
Comment (VaishakMB08) [reply lib/src/web_view_stack.dart]: removed print statement

File: lib/constants/navigation_constant.dart [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class WebViewConstant {
+  static const String facebook = "fb://page/";
```
Comment (nikitharajexathought) [inline lib/constants/navigation_constant.dart:2]: whats the naing concention of constants? constats are always title case :)

File: lib/constants/navigation_constant.dart [hunk 4]
```diff
@@ -0,0 +1,4 @@
+class WebViewConstant {
```
Comment (nikitharajexathought) [inline lib/constants/navigation_constant.dart:1]: name of the file and object should be plural, because we have multiple constants rigght?

**Review by nikitharajexathought — CHANGES_REQUESTED** [review]

File: lib/constants/navigation_constant.dart (hunk 3 above)
Comment (VaishakMB08) [reply lib/constants/navigation_constant.dart:2]: changed the singular file name to plural

File: lib/constants/navigation_constant.dart (hunk 4 above)
Comment (VaishakMB08) [reply lib/constants/navigation_constant.dart:1]: made changes

**Review by nikitharajexathought — APPROVED** [review]

**Pull request merged**